### PR TITLE
 Fixed: generated ng-add-pug-loader.js file had errors

### PR DESCRIPTION
**Pull request by lekhmanrus**
image

### CODE REVIEW

File: src/schematics/ng-add/index.ts [hunk 1]
```diff
@@ -148,5 +148,7 @@ function getPugLoaderRules(): string {
     {
       test: ${partialRegex},
       loader: 'pug-loader'
-    },`.replace(/\s+/gm, ' ');
+    },`
+      .replace(/\s+/gm, ' ')
+      .replace(/^\s/, '');
```
Comment (danguilherme) [inline src/schematics/ng-add/index.ts]: This change is not necessary.
Please focus only on the current problem.

Comment (lekhmanrus) [reply src/schematics/ng-add/index.ts]: I agree, but it looks better without first space (imho).

**Comment by lekhmanrus** [comment]
> Could you let your branch even with `origin`?
> Here's a guide from Github on how to do that: [Syncing a fork](https://help.github.com/articles/syncing-a-fork/).

Sorry, I don't have much experience with this. I read guide, but I don't get what I should do with this pull request now.

**Comment by danguilherme** [comment]
You must update your local code to the code that was changed in this repo after you've forked.

Just follow the following steps and you're good to go:

**Inside your repo folder:**
1. Configure a remote that points to this repository:
   `git remote add upstream https://github.com/danguilherme/ng-cli-pug-loader`
1. Update your remote code:
   `git fetch upstream`
1. Check out your fork's local `master` branch (you're probably in that already)
   `git checkout master`
1. Merge the changes from `upstream/master` into your local master branch. This brings your fork's `master` branch into sync with this repo (the `upstream` repository), without losing your local changes
   `git merge upstream/master`

It will prevent old commits, that were already merged to this repo, from piling up:

image

**Comment by lekhmanrus** [comment]
But I know that I did all these steps before starting work on my local changes yesterday.

**Comment by danguilherme** [comment]
Sorry I'll have to do this myself to ship the fixed version quickly

**Comment by lekhmanrus** [comment]
@danguilherme Thanks!

**Comment by danguilherme** [comment]
Fixed in [v0.1.7](https://github.com/danguilherme/ng-cli-pug-loader/releases/tag/v0.1.7) (e819d9948209f163f8c397c8d321a5ef955135f0).

**Pull request closed without merging**